Removing these macros fixes:

../test/rist/unit/srp_examples.c:20:51: error: expected declaration specifiers or '...' before string constant
   20 | #define calloc(num, size) _test_calloc(num, size, __FILE__, __LINE__)
      |                                                   ^~~~~~~~
../test/rist/unit/srp_examples.c:20:61: error: expected declaration specifiers or '...' before numeric constant
   20 | #define calloc(num, size) _test_calloc(num, size, __FILE__, __LINE__)
      |                                                             ^~~~~~~~
../test/rist/unit/srp_examples.c:21:35: error: expected declaration specifiers or '...' before string constant
   21 | #define free(obj) _test_free(obj, __FILE__, __LINE__)
      |                                   ^~~~~~~~
../test/rist/unit/srp_examples.c:21:45: error: expected declaration specifiers or '...' before numeric constant
   21 | #define free(obj) _test_free(obj, __FILE__, __LINE__)
      |                                             ^~~~~~~~


--- a/test/rist/unit/srp_examples.c
+++ b/test/rist/unit/srp_examples.c
@@ -15,12 +15,6 @@
 #include "src/crypto/srp_constants.h"
 #define DEBUG_USE_EXAMPLE_CONSTANTS 1
 
-#if HAVE_MBEDTLS
-#define malloc(size) _test_malloc(size, __FILE__, __LINE__)
-#define calloc(num, size) _test_calloc(num, size, __FILE__, __LINE__)
-#define free(obj) _test_free(obj, __FILE__, __LINE__)
-#endif
-
 #include "src/crypto/srp.c"
 #include "src/crypto/srp_constants.c"
 
